packages/erxes-ui/src/components/richTextEditor/extensions/Table/Header/Header.ts

```diff
@@ -6,6 +6,9 @@ export const Header = TiptapHeader.extend({
       style: {
         parseHTML: (element) => element.getAttribute('style'),
       },
+      align: {
+        parseHTML: (element) => element.getAttribute('align'),
+      },
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** Review the Header extension for correct handling of alignment attributes.